### ISSUE TITLE
did you abandonned this project ?

### ISSUE THREAD

**Issue by pducharme** [DillonN/NiceHashMiner#30]
Honest question, since profits are not as good as 1 month ago, did you decide to just not work on this fork anymore?  Will the miners out of date ever be updated ?  if not, can you provide some instructions to manually update them ?  Thanks for the good work!

**Comment by DillonN** [comment]
No it's not abandoned, but there probably won't be any updates this week. Keep an eye out next week for a release and update regarding the future of the repo

If you want to update the miners you can extract them to their respective folders, and make sure the executable has the same name. Note that you may get issues since the new versions have not been tested for use in NHM

**Comment by pducharme** [comment]
i'll keep an eye for next week.  just wanted to know if you just abandonned it, glad you did not!!

**Comment by CreamyG31337** [comment]
yeah i'm pretty sure you can't just drop in the latest claymore as they changed some stuff. i mean, the first change is NEW GPU SORTING so I can't imagine that not breaking something.
https://github.com/nanopool/Claymore-Dual-Miner/releases/tag/v9.6
There's like zero performance improvements so why bother.

**Comment by abudfv** [comment]
What about adding algos, supported by Nicehash, such as Kessak and others? Sometimes they are more profitable

**Comment by goddas** [comment]
there is a new 9.7 Claymore miner, please post some instruction on how we can add it or make a new release! Thanks!

**Comment by pducharme** [comment]
I just put claymore Dual 9.7 by replacing the content of claymore_dual in 3rdparty bin folder with the new version. Instant increase from 29 to 32MH/s on my 4 x 1070 and from 20.5 to 22MH/s on my 1060!

**Comment by CreamyG31337** [comment]
@pducharme What's so hard about putting a link to it so we can figure out what you're talking about? 9.7 isn't even on GitHub. Apparently, you downloaded it from here:
https://bitcointalk.org/index.php?topic=1433925.0

**Comment by ASFx2600** [comment]
@pducharme I did the same and it's working great so far. It seems as though -dcri values need to be adjusted for better performance as well. I boosted dagger/lbry up to 200 on my 1080 Ti and getting an even higher has rate compared to when I had it set to 90 previously. I'll take some time to test various dcri values on the 1080 Ti tomorrow.

**Comment by CreamyG31337** [comment]
claymore dual miner 9.7 seems to work fine even on the box I have with both AMD and Nvidia cards. I don't know if others will have the same luck, so I wouldn't suggest just updating without keeping a close eye on it.

**Comment by ydem** [comment]
9.7 works great on my 1070! had to adjust -drci to 20 on mine to get best performance (depends on each card manufacturer), but how do you make the setting work on extra parameters?! i did type '-drci 20' on all dual claymore ones didn't work, also tried '--drci 20' also didn't work, always takes me to the default 30...

**Comment by abudfv** [comment]
ASFx2600 even on 1080 best dcri for lbc is 400+. For 1080ti 500 is the best and definitely could be better with greater value. Of course it depends on price eth/lbc but for now it is so.

**Comment by gpcola** [comment]
On my 980, solo benching ETH is ~5% _worse_ with 9.7 than with 9.5 but ~5% _better_ in dual benches. 2nd coin unchanged in all cases.

**Comment by DillonN** [comment]
CD 9.7 has been added to the latest release https://github.com/DillonN/NiceHashMiner/releases/tag/1.7.6.4
